### DISCUSSION TITLE
Cocoapods support

### DISCUSSION THREAD

**Discussion by AgapovOne** [pointfreeco/swift-custom-dump#44] [Ideas]
We at [aviasales](https://github.com/KosyanMedia) would like to use this great library for our big codebase. We use Cocoapods as a package manager and cannot move easily to SPM even for testing purposes.
Is it okay if we will create a fork [here](https://github.com/KosyanMedia/swift-custom-dump/tree/main) with a custom podspec that references your project?

**Comment by tahirmt** [comment]
We just published to cocoapods in our private podspecs repo without the need of a fork. All you really need is a podfile.

**Comment by mbrandonw** [comment]
Hi @AgapovOne, sorry we did not see this when you first opened the issue. It's very late, but FWIW, it is totally fine to manage your own podspecs and we definitely encourage it!
